riscv/insns/sspush_x1.h

```diff
@@ -0,0 +1,12 @@
+#include "zicfiss.h"
+require_extension(EXT_ZICFISS);
+
+bool xSSE = false;
+SET_XSSE(xSSE);
+
+if (xSSE) {
+  reg_t ssp = STATE.ssp->read();
+  const auto new_ssp = ssp - xlen / 8;
+  store_to_ssp(new_ssp, RA);
+  STATE.ssp->write(new_ssp);
+}
```

Comment:
If we used `RS1` instead of `RA` then we can just include "sspush_x1.h" in "sspush_x5.h"

Reply:
Done